.changeset/lucky-laws-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@docs/storybook": patch
```

Comment:
No need for a changeset since we don't actually release anything :) (change to an empty one).
Docs get published as soon as the PR is merged.

It would be good if we could control when we release the docs instead of it possibly being ahead of our releases, but we haven't really found a way as of yet - definitely known though.